src/tools/context/initial-context.ts

```diff
@@ -17,6 +17,14 @@ Content types define the structure and blueprint for language variants. They spe
 ### Content Type Snippets
 Content type snippets are reusable field groups that promote consistency across multiple content types. Following the DRY principle (define once, use everywhere), one snippet can be used across multiple content types. This prevents duplication and ensures consistency when you need the same fields across different content types.
 
+### Collections
+Collections organize content items into logical groups by team, brand, or project. Each content item belongs to exactly one collection.
+
+### Languages
+Languages define available locales for content. Each language can have a fallback language for content inheritance and can be activated or deactivated.
+
+**CRITICAL**: Only active languages can be modified. To update a deactivated language, first activate it (is_active: true must be the first operation).
```

Comment:
This is a pretty low-level information. Given we are trying to minimize the size of this tool, wouldn't it be enough to have this information in the description of `patch-language-mapi`?